pkg/odo/cli/catalog/describe/service.go

```diff
@@ -33,6 +44,15 @@ type DescribeServiceOptions struct {
 	plans   []svc.ServicePlan
 	// generic context options common to all commands
 	*genericclioptions.Context
+
+	// Operator backed services params
+	// split the name provided on CLI and populate servicetype & customresource
+	isOperator     bool
+	SVCSupported   bool
+	ServiceType    string
```

Comment:
What do `SVCSupported` and `ServiceType` elements of this struct? Can you please add commnet?

Reply:
added comments